ts/screens/onboarding/PinScreen.tsx

```diff
@@ -486,9 +459,7 @@ class PinScreen extends React.PureComponent<Props, State> {
       .catch(e => {
         handleSendAssistanceLog(
           this.choosenTool,
-          `setPin error ${e ? e.toString() : ""}`,
-          TypeLogs.DEBUG,
-          instabuglogTag
+          `setPin error ${e ? e.toString() : ""}`
```

Comment:
```suggestion
          `setPin error ${e?.toString?.() ?? ""}`
```
since we are, a more compress a safer `toString` invoking
Could it be an improvement?